### DISCUSSION TITLE
Install-Module PnP.PowerShell -AllowPrerelease

### DISCUSSION THREAD

**Discussion by diplobelsdb** [pnp/powershell#47] [Q&A] [answered]
PS C:\Windows\system32> Install-Module PnP.PowerShell -AllowPrerelease

Install-Module : A parameter cannot be found that matches parameter name 
'AllowPrerelease'.

Is there anything required to enable the installation of this new PnP.PowerShell module?

**Comment by erwinvanhunen** [comment, marked as answer]
You are most likely running a too old version of PowerShell Get. 

See https://docs.microsoft.com/en-us/powershell/scripting/gallery/installing-psget for more info on how to update.
